### PR TITLE
Ignore things that cannot be known in admission control

### LINKED ISSUE #1796: OriginNamespaceLifecycle must not reject objects not in latest.RESTMapper
It should not be required for 

```
	defaultVersion, kind, err := latest.RESTMapper.VersionAndKindForResource(a.GetResource())
	if err != nil {
		return err
	}
```

to succeed for `pkg/project/admission/admission.go`

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Fixes #1796 

@smarterclayton - is this what you were looking for?

### CODE REVIEW

File: pkg/project/admission/admission.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ type lifecycle struct {
 func (e *lifecycle) Admit(a admission.Attributes) (err error) {
 	defaultVersion, kind, err := latest.RESTMapper.VersionAndKindForResource(a.GetResource())
 	if err != nil {
-		return err
+		return nil
```
Comment (mfojtik) [inline pkg/project/admission/admission.go]: maybe add V(5) info messages if this happen?

Comment (smarterclayton) [inline pkg/project/admission/admission.go]: Yeah, 4 or 5

**Comment by derekwaynecarr** [comment]
Updated PR per logging request, this should be good to go now.

**Comment by derekwaynecarr** [comment]
Is there a reason you chose to close this versus merge it?

Did your desired goal change?

**Comment by smarterclayton** [comment]
I thought this was the issue.

**Comment by smarterclayton** [comment]
Instead, LGTM [merge]

**Comment by derekwaynecarr** [comment]
re [merge]

**Comment by derekwaynecarr** [comment]
Other merged commits caused compile errors, re [merge]

**Pull request merged**